.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,2 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
 connection.project.dir=
 eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Java/jdk-17.0.1
```

Comment:
That shouldn't be here. Hava home path can't be project-specific settings, they can be different for each user.